File: Python/bee 1036.py
import math

a,b,c = map(float, input("").split(" "))
delta = b**2 - (4 * a * c)

if a != 0 and delta >= 0:
    R1 = (-b + math.sqrt(delta))/(2*a)
    R2 = (-b - math.sqrt(delta))/(2*a)
    print(f"R1 = {R1:.5f}\nR2 = {R2:.5f}")
else:
    print("Impossivel calcular")

File: Kotlin/bee 1036.kt
import java.io.BufferedReader
import java.io.InputStreamReader
import kotlin.math.pow
import kotlin.math.sqrt

fun main(args: Array<String>) {
    val br = BufferedReader(InputStreamReader(System.`in`))
    val (a, b, c) = br.readLine().split(" ").map {it.toDouble()}

   val delta = b.pow(2) - (4 * a * c)
    if(delta < 0 || a == 0.0) {
       println("Impossivel calcular")
    } else {
        val r1 = (-b + sqrt(delta)) / (2 * a)
        val r2 = (-b - sqrt(delta)) / (2 * a)

        println(String.format("R1 = %.5f\nR2 = %.5f", r1, r2))
    }

}
